.github/workflows/linting_checks.yml

```diff
@@ -0,0 +1,41 @@
+name: checks
```

Comment:
I thought we decided against the two separate workflows for lint/test?

Reply:
this was already merged which is why I raised the issue #63

Reply:
:+1: